hazelcast/src/main/java/com/hazelcast/collection/impl/queue/QueueService.java

```diff
@@ -93,11 +93,11 @@ public LocalQueueStatsImpl createNew(String key) {
 
     public QueueService(NodeEngine nodeEngine) {
         this.nodeEngine = nodeEngine;
-        ScheduledExecutorService defaultScheduledExecutor
-                = nodeEngine.getExecutionService().getDefaultScheduledExecutor();
+        SchedulingExecutorService defaultSchedulingExecutorService
```

Comment:
Default scheduler ----> I would rename it to the Global Scheduler. The one that is the working horse for all kinds of stuff.